### ISSUE TITLE
`React Native Debugger` is now useless with Expo SDK 49+, since remote debugging is disabled.

### ISSUE THREAD

**Issue by AJTJ** [apollographql/apollo-client#11065]
### Issue Description

Afaik, apollo client users were predominantly depending on `React Native Debugger` to use Expo.

Now that Expo SDK 49 has disabled remote debugging, that no longer seems to be an option, and I haven't found any immediate options for integrating the apolo dev tools to the hermes engine.

Not having access to the apollo dev tools is a huge hinderance.

@phryneas tagging you, as per your request.

### Link to Reproduction

https://blog.expo.dev/expo-sdk-49-c6d398cdf740

**Comment by jasongaare** [comment]
Following since this is a definite pain point on React Native.

We have a bare RN app (not using expo) but I've had relative success installing this (very simple) package to continue to use Apollo Client DevTool in `React Native Debugger` 

The package: https://github.com/gusgard/react-native-devsettings

**Comment by AugustoAleGon** [comment]
> I've just merged #11364 which shows a way of inspecting the Apollo Client cache that at least worked for me when testing.
> 
> I can also connect the chrome debugger to the app (I updated to Expo 49 and it's using Hermes) without problems, so this might have been fixed upstream by now?

How Did you make it work for Chrome Debugger?

**Comment by phryneas** [comment]
@AugustoAleGon no, but with Flipper.

**Comment by 6ase** [comment]
any news about this problem? :(

**Comment by phryneas** [comment]
@6ase yes, you can use the flipper debugger with the Apollo Client plugin like described above.
